realm/src/androidTest/java/io/realm/RealmTest.java

```diff
@@ -190,6 +190,8 @@ public void testGetInstanceClearsCacheWhenFailed() {
 
     public void testInstanceIdForHashCollision() {
         // Ea.hashCode() == FB.hashCode()
+        Realm.deleteRealmFile(getContext(), "Ea");
```

Comment:
No, really. I was using a custom compiled prerelease of Realm Core in my previous run, and forward compatibility shouldn't be a concern.